test/common/performance/receiver/type_extractor.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package receiver
+
+import cloudevents "github.com/cloudevents/sdk-go"
+
+type TypeExtractor func(event cloudevents.Event) string
+
+func EventTypeExtractor(event cloudevents.Event) string {
+	return event.Type()
```

Comment:
Do we anticipate other ways to extract a type?

Reply:
https://github.com/knative/eventing/pull/1979#discussion_r333362955